### PR TITLE
Add support for Probot Stale

### LINKED ISSUE #529: Use stale bot for inactive issues
We have many issues that are long running / did not get any attention. 

AFAIK we should close them after a while. If it's still a problem for anyone, it should come back or someone should be working on this.

[Stale](https://probot.github.io/apps/stale/) could close this issues for us, so we can focus on active issues.

### PR DESCRIPTION
**Pull request by phansys**
Closes #529.

### CODE REVIEW

File: project/.github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
```
Comment (OskarStark) [inline project/.github/stale.yml]: excePt?

File: project/.github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
```
Comment (core23) [inline project/.github/stale.yml]: Keep in mind, this will close nearly all issues...

Comment (core23) [reply project/.github/stale.yml]: Maybe we should start with a higher value first... 180 days?

Comment (OskarStark) [reply project/.github/stale.yml]: Yes why not

**Comment by core23** [comment]
We should keep one config inside the `dev-kit` and use `_extends` in the other repositories:

https://probot.github.io/docs/best-practices/#store-configuration-in-the-repository

File: project/.github/stale.yml (hunk 1 above)
Comment (phansys) [reply project/.github/stale.yml]: Nope, this is "[exempt](https://github.com/probot/stale/blob/cd1e7623c6971731d4c19846d62b4dd383616171/lib/stale.js#L68)".

**Comment by core23** [comment]
@soullivaneuh Can you merge this and install stale for us? https://probot.github.io/apps/stale/

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - critical
```
Comment (core23) [inline .github/stale.yml]: Should we add an other label like `no stale` to keep specific issues open?

Comment (phansys) [reply .github/stale.yml]: Yes, or `wont stale` (to be consistent with `wont fix`).
Let me know and I'll be pushing again.

Comment (core23) [reply .github/stale.yml]: Or something like `keep open` or just `keep`

Comment (phansys) [reply .github/stale.yml]: Done!

File: .github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - keep
```
Comment (core23) [inline .github/stale.yml]: You should also update this file: https://github.com/sonata-project/dev-kit/blob/master/config/dev-kit.yml

Comment (phansys) [reply .github/stale.yml]: It's updated now.

File: .github/stale.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
```
Comment (soullivaneuh) [inline .github/stale.yml]: The configuration labels are enough explicit, I think the comments can be removed from this file.

Comment (phansys) [reply .github/stale.yml]: It's ok. The comments were removed.

**Comment by OskarStark** [comment]
Is this FOR devkit or for the projects?

**Comment by core23** [comment]
> Is this FOR devkit or for the projects?

both

**Comment by core23** [comment]
After merging this, we only got ~150 open issues :rocket: 

https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+org%3Asonata-project+archived%3Afalse+updated%3A%3E2019-06-01+

**Comment by OskarStark** [comment]
Ok but in the repo we end up with a file which extends something which isn’t in the repo right? Only files in /project dir will be added by a PR

**Comment by core23** [comment]
> Ok but in the repo we end up with a file which extends something which isn’t in the repo right? Only files in /project dir will be added by a PR

That's right the `dev-kit` will put a simple config inside `all repos` and that file will extend the file located inside the `dev-kit` repo.

**Comment by phansys** [comment]
> Ok but in the repo we end up with a file which extends something which isn’t in the repo right? Only files in /project dir will be added by a PR

Yes, based on the leveraging of the feature proposed here: https://github.com/sonata-project/dev-kit/pull/530#issuecomment-564880897.

>Just put common configuration keys in a repository within your organization. Then reference this repository from config files with the same name.

**Pull request merged**